backend/controllers/loginController.go

```diff
@@ -164,4 +165,47 @@ func (l loginController) PasswordReset(c *gin.Context) {
 		"resetCode": resetCode,
 	})
 	return
+
+}
+
+func (l loginController) PasswordReset(c *gin.Context) {
+	email := c.Param("email")
+	resetCode := c.Param("resetcode")
```

Comment:
Wouldn't it make more sense to have the data coming in the request body instead of as parameters? For simpler routes its fine but it seems for a route that needs three items it becomes a little unwieldy to not just have the data sent as json.